src/test/graphql/queries/pocketSave.integration.ts

```diff
@@ -0,0 +1,183 @@
+import { readClient } from '../../../database/client';
+import { ContextManager } from '../../../server/context';
+import { startServer } from '../../../server/apollo';
+import { Express } from 'express';
+import { ApolloServer } from '@apollo/server';
+import request from 'supertest';
+
+describe('getPocketSaveByItemId', () => {
+  const db = readClient();
+  const headers = { userid: '1' };
+  const date1 = new Date('2008-10-21 13:57:01');
+  const date2 = new Date('0000-00-00 00:00:00');
+  const date3 = new Date('2008-10-21 14:00:01');
+  const date4 = new Date('2012-08-13 15:32:05');
+  const date5 = new Date('2008-11-03 08:51:01');
+  let app: Express;
+  let server: ApolloServer<ContextManager>;
+  let url: string;
+
+  const GET_POCKET_SAVE = `
```

Comment:
I know this isn't happening in every repo, but I would recommend making this a `gql` tag and using the graphql print function to get these converted back to strings. Imports for these if you haven't seen them:

```
import { gql } from 'graphql-tag';
import { print } from 'graphql';
```

It's nice for us just semantically to be able to find all of our queries in a repo by searching for a string, it's nice to get syntax highlighting, and we're likely going to need them if we ever want to introduce graphql query linting without it being a huge pain in the butt later.

Reply:
agree, that's the pattern we follow in other repo, e.g here: https://github.com/Pocket/user-api/blob/1b63de9feafa18772d6f74cdc27426a2b384f819/src/test/functional/context.integration.ts#L46

Reply:
makes sense. a lot of this is copypasta from the existing setup for pre-existing code, but going to try to rework now to follow that setup.